snippets/csharp/VS_Snippets_CLR_System/system.Random.Next/CS/next3.cs

```diff
@@ -1,10 +1,10 @@
-﻿// <Snippet5>
-using System;
+﻿using System;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet5>
       Console.Write("Number of random numbers to generate: ");
```

Comment:
I tried this, and the current try.net experience for `Console.Readline` is not great.

@aprilspeight: Will that change with the updates coming this sprint?

@jonsequitur @LadyNaggaga  What experience should we expect from the underlying platform?

Reply:
@BillWagner engineering believes that there's a bug (i.e. keeps asking for input). However, this bug is not something that was within the scope of the scaffolding feature to resolve. Scaffolding code with Console.Readline works though.

Reply:
@BillWagner, @aprilspeight -- for now, I removed the Readline-related statements & this sample simply generates 10 numbers.